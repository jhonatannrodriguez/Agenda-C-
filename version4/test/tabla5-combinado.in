# creo una tabla de tamanio 15
crearTTablaPersonas 15

# creo 15 personas, las inserto en la tabla, verifico que pertenezcan y las obtengo

crearPersona 1 11 Persona1
insertarPersonaEnTabla
perteneceATTablaPersonas Persona1
obtenerPersonaDeTTablaPersonas Persona1
crearPersona 2 22 Persona2
insertarPersonaEnTabla
perteneceATTablaPersonas Persona2
obtenerPersonaDeTTablaPersonas Persona2
crearPersona 3 33 Persona3
insertarPersonaEnTabla
perteneceATTablaPersonas Persona3
obtenerPersonaDeTTablaPersonas Persona3
crearPersona 4 44 Persona4
insertarPersonaEnTabla
perteneceATTablaPersonas Persona4
obtenerPersonaDeTTablaPersonas Persona4
crearPersona 5 55 Persona5
insertarPersonaEnTabla
perteneceATTablaPersonas Persona5
obtenerPersonaDeTTablaPersonas Persona5
crearPersona 6 66 Persona6
insertarPersonaEnTabla
perteneceATTablaPersonas Persona6
obtenerPersonaDeTTablaPersonas Persona6
crearPersona 7 77 Persona7
insertarPersonaEnTabla
perteneceATTablaPersonas Persona7
obtenerPersonaDeTTablaPersonas Persona7
crearPersona 8 88 Persona8
insertarPersonaEnTabla
perteneceATTablaPersonas Persona8
obtenerPersonaDeTTablaPersonas Persona8
crearPersona 9 99 Persona9
insertarPersonaEnTabla
perteneceATTablaPersonas Persona9
obtenerPersonaDeTTablaPersonas Persona9
crearPersona 10 100 Persona10
insertarPersonaEnTabla
perteneceATTablaPersonas Persona10
obtenerPersonaDeTTablaPersonas Persona10
crearPersona 11 110 Persona11
insertarPersonaEnTabla
perteneceATTablaPersonas Persona11
obtenerPersonaDeTTablaPersonas Persona11
crearPersona 12 120 Persona12
insertarPersonaEnTabla
perteneceATTablaPersonas Persona12
obtenerPersonaDeTTablaPersonas Persona12
crearPersona 13 130 Persona13
insertarPersonaEnTabla
perteneceATTablaPersonas Persona13
obtenerPersonaDeTTablaPersonas Persona13
crearPersona 14 140 Persona14
insertarPersonaEnTabla
perteneceATTablaPersonas Persona14
obtenerPersonaDeTTablaPersonas Persona14
crearPersona 15 150 Persona15
insertarPersonaEnTabla
perteneceATTablaPersonas Persona15
obtenerPersonaDeTTablaPersonas Persona15

# imprimo la tabla en el orden que quedaron insertados
imprimirTTablaPersonas


# elimino las personas de id par y verifico que no pertenecen
eliminarPersonaDeTTablaPersonas Persona2
perteneceATTablaPersonas Persona2
eliminarPersonaDeTTablaPersonas Persona4
perteneceATTablaPersonas Persona4
eliminarPersonaDeTTablaPersonas Persona6
perteneceATTablaPersonas Persona6
eliminarPersonaDeTTablaPersonas Persona8
perteneceATTablaPersonas Persona8
eliminarPersonaDeTTablaPersonas Persona10
perteneceATTablaPersonas Persona10
eliminarPersonaDeTTablaPersonas Persona12
perteneceATTablaPersonas Persona12
eliminarPersonaDeTTablaPersonas Persona14
perteneceATTablaPersonas Persona14

# imprimo la tabla 
imprimirTTablaPersonas

# agrego dos colisiones a la tabla
crearPersona 100 1100 Pers1ona
insertarPersonaEnTabla
perteneceATTablaPersonas Pers1ona
obtenerPersonaDeTTablaPersonas Pers1ona
crearPersona 700 7700 Pers7ona
insertarPersonaEnTabla
perteneceATTablaPersonas Pers7ona
obtenerPersonaDeTTablaPersonas Pers7ona

# elimino dos elementos que colisionan y verifico que no pertenecen
eliminarPersonaDeTTablaPersonas Pers1ona
perteneceATTablaPersonas Pers1ona
eliminarPersonaDeTTablaPersonas Persona7
perteneceATTablaPersonas Persona7

# libero la tabla
liberarTTablaPersonas

Fin