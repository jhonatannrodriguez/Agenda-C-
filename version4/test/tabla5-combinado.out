1>#  creo una tabla de tamanio 15.
2>3>#  creo 15 personas, las inserto en la tabla, verifico que pertenezcan y las obtengo.
4>5>6>La persona con nombre Persona1 está en la tabla.
7>Persona 1: Persona1, 11 años
8>9>10>La persona con nombre Persona2 está en la tabla.
11>Persona 2: Persona2, 22 años
12>13>14>La persona con nombre Persona3 está en la tabla.
15>Persona 3: Persona3, 33 años
16>17>18>La persona con nombre Persona4 está en la tabla.
19>Persona 4: Persona4, 44 años
20>21>22>La persona con nombre Persona5 está en la tabla.
23>Persona 5: Persona5, 55 años
24>25>26>La persona con nombre Persona6 está en la tabla.
27>Persona 6: Persona6, 66 años
28>29>30>La persona con nombre Persona7 está en la tabla.
31>Persona 7: Persona7, 77 años
32>33>34>La persona con nombre Persona8 está en la tabla.
35>Persona 8: Persona8, 88 años
36>37>38>La persona con nombre Persona9 está en la tabla.
39>Persona 9: Persona9, 99 años
40>41>42>La persona con nombre Persona10 está en la tabla.
43>Persona 10: Persona10, 100 años
44>45>46>La persona con nombre Persona11 está en la tabla.
47>Persona 11: Persona11, 110 años
48>49>50>La persona con nombre Persona12 está en la tabla.
51>Persona 12: Persona12, 120 años
52>53>54>La persona con nombre Persona13 está en la tabla.
55>Persona 13: Persona13, 130 años
56>57>58>La persona con nombre Persona14 está en la tabla.
59>Persona 14: Persona14, 140 años
60>61>62>La persona con nombre Persona15 está en la tabla.
63>Persona 15: Persona15, 150 años
64>#  imprimo la tabla en el orden que quedaron insertados.
65>Persona 10: Persona10, 100 años
Persona 4: Persona4, 44 años
Persona 11: Persona11, 110 años
Persona 5: Persona5, 55 años
Persona 12: Persona12, 120 años
Persona 6: Persona6, 66 años
Persona 13: Persona13, 130 años
Persona 7: Persona7, 77 años
Persona 14: Persona14, 140 años
Persona 8: Persona8, 88 años
Persona 15: Persona15, 150 años
Persona 9: Persona9, 99 años
Persona 1: Persona1, 11 años
Persona 2: Persona2, 22 años
Persona 3: Persona3, 33 años
66>#  elimino las personas de id par y verifico que no pertenecen.
67>68>La persona con nombre Persona2 NO está en la tabla.
69>70>La persona con nombre Persona4 NO está en la tabla.
71>72>La persona con nombre Persona6 NO está en la tabla.
73>74>La persona con nombre Persona8 NO está en la tabla.
75>76>La persona con nombre Persona10 NO está en la tabla.
77>78>La persona con nombre Persona12 NO está en la tabla.
79>80>La persona con nombre Persona14 NO está en la tabla.
81>#  imprimo la tabla .
82>Persona 11: Persona11, 110 años
Persona 5: Persona5, 55 años
Persona 13: Persona13, 130 años
Persona 7: Persona7, 77 años
Persona 15: Persona15, 150 años
Persona 9: Persona9, 99 años
Persona 1: Persona1, 11 años
Persona 3: Persona3, 33 años
83>#  agrego dos colisiones a la tabla.
84>85>86>La persona con nombre Pers1ona está en la tabla.
87>Persona 100: Pers1ona, 1100 años
88>89>90>La persona con nombre Pers7ona está en la tabla.
91>Persona 700: Pers7ona, 7700 años
92>#  elimino dos elementos que colisionan y verifico que no pertenecen.
93>94>La persona con nombre Pers1ona NO está en la tabla.
95>96>La persona con nombre Persona7 NO está en la tabla.
97>#  libero la tabla.
98>99>Fin.
